templates/navbar.html

```diff
@@ -98,6 +98,7 @@
               </li>
               {% endif %}
               <li role="separator" class="divider"></li>
+              <li><a href="/accounts/password/change/">Change Password</a>{{environment.}}</li>
```

Comment:
Is that period in environment supposed to be there? Also where does that variable come from? @gregelin

Reply:
I think `{{environment.}}` is accidental debug/dev code I failed to delete.

Reply:
okay i'll delete it directly.